pydocstringformatter/utils/argument_parsing.py

```diff
@@ -80,3 +99,37 @@ def _parse_toml_file(
             arguments += _parse_toml_option(key, value)
 
         parser.parse_args(arguments, namespace)
+
+
+def _load_default_formatters(
```

Comment:
```suggestion
def _load_formatters_default_option(
```

If we can indeed pass non-standard formatter this is not longer only `default`.